server/sonar-server/src/main/java/org/sonar/server/user/ws/UserJsonWriter.java

```diff
@@ -93,7 +93,7 @@ public void write(JsonWriter json, @Nullable UserDto user) {
   }
 
   private void writeGroupsIfNeeded(JsonWriter json, Collection<String> groups, @Nullable Collection<String> fields) {
```

Comment:
indeed the class is not unit tested but it is covered by tests of callers (DeactivateActionTest, SearchActionTest, UpdateActionTest, ...). Not the best design still.